main.go

```diff
@@ -2,20 +2,18 @@ package main
 
 import (
 	"log"
-	"os"
-
-	"github.com/xlab/closer"
 
+	"github.com/ivanterekh/go-skeleton/env"
 	"github.com/ivanterekh/go-skeleton/server"
 )
 
+type config struct {
```

Comment:
I think we don't need this structure anymore

Reply:
Removed.